airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -642,6 +648,51 @@ def _flat_associations(self, records: Iterable[MutableMapping]) -> Iterable[Muta
             yield record
 
 
+class AssociationsStream(Stream):
```

Comment:
Not necessarily something to fix now, but I was confused by this line because `Stream` usually refers to the the ABC class defined in core.py

Reply:
yep that's confusing but I'd prefer to fix it in a separate PR